test-fixtures/ci/update_bitrise.json

```diff
@@ -0,0 +1,59 @@
+"blocks": [
+		{
+			"type": "section",
+			"text": {
+				"type": "plain_text",
+				"text": "🚨 *GitHub Action Failed* 🚨",
```

Comment:
We may need to format this to align with other notifications but that can be done in a follow up